tests/tdscf-3/input.dat

```diff
@@ -0,0 +1,43 @@
+# Excitation energies
+
+rhf_wB97X_RPA_cc_pvdz = [0.24721269233329,
+                         0.31319977578113,
+                         0.34444772468718,
+                         0.41038703401580,
+                         0.45278105243050,
+                         0.56744628843739,
+                         0.71909971752194,
+                         0.75183601398849,
+                         0.82952408171446,
+                         0.84003438794418]
+
+
+
+
+molecule water{
+0 1
+
+O           0.000000    0.000000    0.135446
+H          -0.000000    0.866812   -0.541782
+H          -0.000000   -0.866812   -0.541782
+no_reorient
+no_com
+symmetry c1
+}
+
+set {
+    reference rhf
+    scf_type pk
+    e_convergence 8
+    d_convergence 8
+    TDSCF_STATES_PER_IRREP [10]
+}
+
+
+e, wfn = energy('wb97x/cc-pvdz', return_wfn=True)
+tdscf(wfn)
```

Comment:
be good to have a test that's one-line `energy('td-wb97x/cc-pvdz')`

Reply:
So I've got `tdscf(wfn)` and `tdscf('wb97x/cc-pvdz')` working. I don't have any handling of functionals with "td-" prefixes, nor is there any one-liner to get tdscf via an `energy` call (the one-liner is a `tddft()` call). Should I add this?

Reply:
So of the three sequences below, I'm in favor of first and third and don't care for second.
* `e, wfn = energy(wb97x/cc-pvdz', return_wfn=True)  \n tdscf(wfn)`
* `tdscf('wb97x/cc-pvdz')`
* `energy('td-wb97x/cc-pvdz')`

In order to get the third, I'd go into proc_table.py and add to the `procedures` dict akin to how the DFT fctls themselves are added, so after any excluding logic for DH or whatever, `procedures['energy']['td-' + key] = proc.run_tdscf` . And then in proc.py, define `def run_tdscf` as essentially the `run_scf()` followed by tdscf(wfn)`

Reply:
I'm not a huge fan of the third. I think its overloading `energy` too much. I think the first two are sufficient.

How do you handle `tdscf(wfn)` with regard to JK saving?

Reply:
Not wanting to overload `energy` too much is a fair point.
Although a `TD-` prefix would go along with the `EOM-` prefix we already have for CCSD, CC2/3. Also how would we want to run a TD-DFT gradient in the future? If we would do `gradient('td-pbe')` then `energy('td-pbe')` seems more natural.

Reply:
The `SAVE_JK` option is set to true in `run_scf` if the `TDSCF_STATES_PER_IRREP` option is set to compute any number of states. I suppose this would break if the states option was set after the initial scf call.

Reply:
I think we must have the third. Most users have no notion how tddft internally different from cis or eom-ccsd, and it would seem strange to them that there's a psi method not accessed through `energy()`. Good point that we have mp2 defined as a fctl -- if it wasn't already excluded from `td-`ing by being a DH, I agree it should be manually removed.